plugin/cmd/kube-scheduler/app/server.go

```diff
@@ -85,6 +85,8 @@ func Run(s *options.SchedulerServer) error {
 	if err != nil {
 		return fmt.Errorf("unable to get hostname: %v", err)
 	}
+	glog.V(3).Info("Initiating leader election process to startup the scheduler server.")
```

Comment:
All the logs should be as close to the same format as possible. I like both possibilities, either a message ending with ``...`` or just with a dot. Though, ``...`` looks like something is happening so it is more user friendly.